tests/commands/load/test_load_cnv_report_cmd.py

```diff
@@ -35,6 +35,7 @@ def test_invalid_path_load_cnv_report(mock_app, case_obj):
         cli,
         ["load", "cnv-report", case_obj["_id"], "invalid-path", "-u"],
     )
-
-    assert "Path 'invalid-path' does not exist." in result.output
+    error_string1 = "Path 'invalid-path' does not exist."
```

Comment:
What about omitting the part of the statement with the quotes and leave just `does not exist`. But I'm afraid that more tests will fail for you after this because I remember clearly that some time ago when we unblocked the Click dependency we modified many tests to be up to date with a more recent version of Click, so I think the easiest would be for you to uninstall and reinstall Click, or just install Scout on a clean environment

Reply:
Fixed!